components/molecule/phoneInput/src/index.js

```diff
@@ -82,15 +85,27 @@ export default function MoleculePhoneInput({
           </AtomIcon>
         </div>
         <div className={`${baseClass}-input-phoneContainer`}>
-          <p className={`${baseClass}-input-prefix-code`}>
-            {selectedPrefix.countryCode}
-          </p>
+          {selectedPrefix && (
+            <p className={`${baseClass}-input-prefix-code`}>
+              {selectedPrefix.countryCode}
+            </p>
+          )}
           <AtomInput
             value={value.toString()}
-            mask={selectedPrefix.mask || '000 000 000'}
+            mask={
+              selectedPrefix && {
```

Comment:
Can we improve the readability of those lines? Maybe, creating a const before and use it here?